.github/workflows/main.yaml

```diff
@@ -134,9 +134,13 @@ jobs:
           target = "${{ matrix.target }}"
           EOF
         if: matrix.target
+      - name: Set sccache port
+        run: |
+          echo "SCCACHE_SERVER_PORT=$RANDOM" >> $GITHUB_ENV
```

Comment:
I think this might cause CI to be flakey, failing at random.  Perhaps we can try to start the server in a loop or something, loop N times (the number of instances we have, and try port `X + i`

Alternatively, we may be able to set env vars with a unique port for each runner on the aarch64 machine